packages/firestore/src/local/shared_client_state.ts

```diff
@@ -168,6 +172,11 @@ export interface SharedClientState {
 
   /** Changes the shared online state of all clients. */
   setOnlineState(onlineState: OnlineState): void;
+
+  writeSequenceNumber(sequenceNumber: ListenSequenceNumber): void;
+  setSequenceNumberListener(
+    cb: (sequenceNumber: ListenSequenceNumber) => void
+  ): void;
```

Comment:
Please take a look at how the remaining "syncers" are set up in the top of this interface.